src/main/java/fitr/Fitr.java

```diff
@@ -16,18 +18,22 @@
     private ExerciseList exerciseList;
     private User user;
 
-    public Fitr(String filePathOfUserConfig, String filePathOfFoodList, String filePathOfExerciseList) {
+    public Fitr(String filePathOfUserConfig, String filePathOfFoodList, String filePathOfExerciseList,
+                String filePathOfTipList) {
         try {
             user = new User();
-            Ui.printGreetingMessage();
-            storage = new Storage(filePathOfUserConfig, filePathOfFoodList, filePathOfExerciseList);
+            storage = new Storage(filePathOfUserConfig, filePathOfFoodList, filePathOfExerciseList, filePathOfTipList);
             if (!storage.readUserConfigFile(user)) {
                 user.setup();
                 storage.writeUserConfigFile(user);
             }
-            Ui.printSuggestQuestion();
             foodList = new FoodList(storage.loadFoodList());
             exerciseList = new ExerciseList(storage.loadExerciseList());
+            TipList tipList = new TipList(storage.loadTipList());
+            Command tipOfTheDay = new TipCommand(tipList);
+            tipOfTheDay.execute(foodList, exerciseList, storage, user);
```

Comment:
Feels like this can be simplified, since the tips will be shown when the program first runs, and the user isn't going to be typing a command for tips. See the comment in the `TipCommand` class..